bootstrap/config/gcp_prototype.yaml

```diff
@@ -0,0 +1,44 @@
+# Sample config for kubeflow bootstrapper
+---
+# Apps that always apply
```

Comment:
Why are we creating multiple apps? Why wouldn't we just create a single app and use the ConfigMap to specify the app we want to create?